packages/app-bridge-app/.eslintrc.cjs

```diff
@@ -32,15 +32,16 @@ module.exports = {
         {
             files: ['*.ts', '*.tsx', '*.mts', '*.cts'],
             rules: {
+                '@typescript-eslint/no-redundant-type-constituents': 'off',
                 'no-prototype-builtins': 'warn',
                 '@typescript-eslint/no-explicit-any': 'warn',
                 '@typescript-eslint/no-floating-promises': 'warn',
                 '@typescript-eslint/no-misused-promises': 'warn',
                 '@typescript-eslint/no-unsafe-argument': 'warn',
-                '@typescript-eslint/no-unsafe-assignment': 'warn',
-                '@typescript-eslint/no-unsafe-call': 'warn',
-                '@typescript-eslint/no-unsafe-member-access': 'warn',
-                '@typescript-eslint/no-unsafe-return': 'warn',
+                '@typescript-eslint/no-unsafe-assignment': 'off',
+                '@typescript-eslint/no-unsafe-call': 'off',
+                '@typescript-eslint/no-unsafe-member-access': 'off',
+                '@typescript-eslint/no-unsafe-return': 'off',
```

Comment:
Was it for development only, or do we want to turn these warnings off for everyone?

Reply:
yeeh i removed them from this repository, as it doesnt create any good feedback but a lot of noise.